website/docs/cloud-docs/vcs/bitbucket-server.mdx

```diff
@@ -110,9 +110,19 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
 
    If this results in a 500 error, it usually means Terraform Cloud was unable to reach your Bitbucket Server instance.
 
-1. Click the "Allow" button. This returns you to Terraform Cloud to enter a SSH key.
+1. Click the "Allow" button. This returns you to Terraform Cloud to configure Advanced Settings.
 
-## Step 3: On Workstation: Create an SSH Key for Terraform Cloud
+## Step 3: On Terraform Cloud, Configure VCS Provider Scope (Optional)
```

Comment:
Adding a new step in before mandatory configuration of SSH Key for Btibucket Server